### ISSUE TITLE
rbh-report csv header

### ISSUE THREAD

**Issue by ysagon** [cea-hpc/robinhood#16]
Hello, there is an inconsistency in the output of rbh-report.

When doing rbh-report --user-info=xxx, the header is the following:

`user      ,     type,      count,        spc_used,        avg_size,`

The problem is on the first field, the comma is surrounded by spaces, it's not the case with the other fields. This makes it hard to parse with python csv Dictreader for example

**Comment by caljess599** [comment]
This is only one of my issues with rhb-report. It also seems to lack a flag to control the output units, which makes scripted math very difficult.

**Comment by tl-cea** [comment]
rbh-du -u <user> almost provide the same info as rbh-report -u
Perhaps it will give you more satisfaction regarding options and outputs.

**Comment by caljess599** [comment]
The floating space after 'user' should be considered a bug fix. Please consider correcting it! --Jessica

**Comment by tl-cea** [comment]
One first step to your proposals:
I submitted a patch to rigth-align the first field like other fields:
    https://review.gerrithub.io/#/c/278620/
It will be part of incoming 3.0 release.
